src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -646,6 +651,10 @@ public Person createPerson(long personSeed, Map<String, Object> demoAttributes)
     // Initialize person.
     Person person = new Person(personSeed);
     person.populationSeed = this.options.seed;
+
+    // Check if we need to override demographics based on preferred provider setting
+    ProviderFinderPreferOne.overrideDemographicsIfPreferredProvider(demoAttributes);
+
```

Comment:
Yeah, I don't actually like how this works, unless there is some reasoning that I'm not understanding.

If you really want to do this, then you should override the `options.city` above in the same way you override the `options.state`.

Reply:
uses options.city and options.state in Generator.java in the init() function as an alternative to setting the default.